dataplane-rpc/dataplane_grpc.proto

```diff
@@ -0,0 +1,128 @@
+syntax = "proto3";
```

Comment:
@sergeymatov you have a duplicate, could you please only one and let's have it please have it in a dir `proto` and name file just `dataplane.proto` for simplicity

Reply:
done